x-pack/plugins/apm/server/lib/services/get_services/get_legacy_data_status.ts

```diff
@@ -24,7 +25,10 @@ export async function getLegacyDataStatus(setup: Setup) {
         size: 0,
         query: {
           bool: {
-            filter: [{ range: { [OBSERVER_VERSION_MAJOR]: { lt: 7 } } }],
+            filter: [
+              { range: { [OBSERVER_VERSION_MAJOR]: { lt: 7 } } },
+              ...rangeQuery(setup.start, setup.end),
```

Comment:
Can we destructure these from `setup` above? Doesn't matter but it's done consistently everywhere else.

Reply:
Will do 👍🏻